### ISSUE TITLE
Windows 10 cannot open todo.txt_cli-2.12.0.zip

### ISSUE THREAD

**Issue by brianminsk** [todotxt/todo.txt-cli#318]
**Do you want to request a *feature* or report a *bug*?**
bug

**What is the current behavior?**
Windows 10 gives an error trying to open todo.txt_cli-2.12.0.zip. File cannot be opened.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

1. Download todo.txt_cli-2.12.0.zip from https://github.com/todotxt/todo.txt-cli/releases. 
2. Click on the file to open it.
3. Windows Explorer gives error: 
```
"Windows cannot open the folder.
 The Compressed (zipped) Folder <path to downloaded zip file> is invalid."
```

**What is the expected behavior?**

**Which versions todo.sh are you using?**
> Run `todo.sh -V`

**Which Operating System are you using?**
Windows 10

**Which version of bash are you using?**
> Run `bash --version`

**Comment by LonghornRach** [comment]
When I navigate to it and try to extract, it says there are no files in it.

**Comment by karbassi** [comment]
Great catch @brianminsk and @LonghornRach. I've fixed this in #319 and reuploaded the zip file.
